### PR TITLE
Added ManagerMenuInitializer

### PR DESCRIPTION
**Pull request by FunnyBytes**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Espress0JavaTeam/AirLine/pull/7" target="_blank">ReviewNinja

File: src/com/acme/view/ManagerMenuInitializer.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package com.acme.view;
```
Comment (VolodymyrBondarchuk) [inline src/com/acme/view/ManagerMenuInitializer.java]: Please add rest clases

Comment (FunnyBytes) [inline src/com/acme/view/ManagerMenuInitializer.java]: Fixed all comments

File: src/com/acme/localisation/Menu/MenuOptions.java [hunk 2]
```diff
@@ -1,23 +1,7 @@
...
-            "[3] Show Possible Destination",
-            "[4] Company Management",
-            "[0] Quit"
-    };
-
-    public static final String[] CLIENT_MAIN_MENU = {
-            "[1] Buy a ticket",
-            "[2] Show Flight Schedule",
-            "[3] Show Possible Destination",
-            "[0] Quit"
```
Comment (VolodymyrBondarchuk) [inline src/com/acme/localisation/Menu/MenuOptions.java]: Why did you delete client menu? As far as I know it is Dima's last piece of code and it is necessary for further development

File: src/com/acme/command/impl/plane/PlaneManagement.java [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-import com.acme.view.MenuInitializer;
+import com.acme.view.ManagerMenuInitializer;
 
 public class PlaneManagement implements Command {
 
     @Override
     public void execute() {
-        MenuInitializer.initPlaneMenu();
-        MenuInitializer.planeMenu.run();
+        ManagerMenuInitializer.initPlaneMenu();
```
Comment (VolodymyrBondarchuk) [inline src/com/acme/command/impl/plane/PlaneManagement.java]: Why do we need this cycled/looped dependencies?  I mean this: PlaneManagement calls ManagerMenuInitializer and ManagerMenuInitializer calls PlaneManagement

Comment (FunnyBytes) [inline src/com/acme/command/impl/plane/PlaneManagement.java]: We need it for implementation of CommandPattern. ManagerMenuInitializer doesn't calls PlaneManagment. In initMainMenu we create Map<Integer, Command> and put there our command PlaneManagment.

Comment (VolodymyrBondarchuk) [inline src/com/acme/command/impl/plane/PlaneManagement.java]: It is looks like spaghetti code. Refactoring will be needed

Comment (FunnyBytes) [inline src/com/acme/command/impl/plane/PlaneManagement.java]: OK, will try to refactor.

**Pull request merged**